### ISSUE TITLE
determining oncogenecity

### ISSUE THREAD

**Issue by ahwagner** [cancervariants/metakb#12]
There have been previous mentions of the need to determine oncogenicity from variant + disease contexts:

https://github.com/cancervariants/metakb/issues/3#issuecomment-443148080
https://github.com/cancervariants/metakb/issues/4#issuecomment-446761789

We should consider approaches to determining when a variant qualifies as "oncogenic" for purposes of matching interpretations as well as a general attribute of a variant + disease.

**Comment by ahwagner** [comment]
@DavidTamborero notes that he has just implemented something to this effect for the Cancer Core Europe clinical reports, he is interested in sharing down the road.

**Comment by gaberudy** [comment]
Yea, I am implementing something similar to this in VSClinical as well using a lot of the components I built for auto-scoring the ACMG guidelines.

Funny, I thought I had invented the word "Oncogencity"! But I'm glad it's in use more broadly than just my products :)

Although I'm at the start of the implementation process, I have a rough roadmap of how I plan on computing oncogenicity. I'd be happy to share / compare notes. Probably would be best in a presentation mode where it can be visual-rich.

**Comment by ahwagner** [comment]
Quoting @jeremywarner from discussion in [#4](https://github.com/cancervariants/metakb/issues/4#issuecomment-447021578):

> This could be of interest: https://www.ncbi.nlm.nih.gov/pubmed/29961819

**Comment by bpitel12** [comment]
Just listened to a webinar yesterday on VarSeq and how information is being pulled from COSMIC, ClinVar, CIViC- very nice! I'm wondering if @gaberudy  and @DavidTamborero should present as a series "Determining Oncogenicity in Rare Variants" to the larger VICC-VTB group at some point where we can talk about what is offered and being developed currently as well as how the VICC group can plan to integrate these ideas into the VTB space.

**Comment by DavidTamborero** [comment]
indeed the interpretation of the 'functional' relevance of the variants needs to be coupled to the exploration of their actionability, since the second needs the the first in many cases and also because the first can use the the second

**Comment by gaberudy** [comment]
@bpitel12 thanks for the mention, and I would happy to give such a presentation.

It turns out we are right in the thick of this, and every day we are making judgement cals about (what if we see a LoF variant in a TSG that has no LoF observed variants in COSMIC? Seems like LoF may not be a driver for this tumor supressor. What if we see a LoF in a oncogene that does have LoF activating mutations? (we have examples of those).

@DavidTamborero my thoughts is that the rare / not yet fully clinically studied variants need a chance to show up on the radar with strong oncogenicity score even if there is not a lot of clinical evidence (case studies, papers, FDA studies, basically stuff in CIViC etc). But of course being well classified in those databases should give a boost.

Let me know if we should plan a presentation, as I would definitely like feedback and standardization around what I'm doing!

**Comment by gaberudy** [comment]
@DavidTamborero Yes, by LoF I mean the sequence ontology class of frameshift, stop gains and canonical splice site disruptions (although we are investigating differentiating between donor splice site ablation where you expect a total protein loss or truncation versus an acceptor splice site ablation which for smaller in-frame exons may result in a functional protein and can act as a driver mutation in an oncogene!).

Actually, two weeks would be a good time for us to present on our "oncogonicity scoring ruberic" as well, lets shoot for trying to do something in the last week of Jan or first week of Feb!
